modules/common/src/main/java/com/sygic/modules/common/MapFragmentWrapper.kt

```diff
@@ -104,15 +102,17 @@ abstract class MapFragmentWrapper : MapFragment(), SdkInitializationManager.Call
         getMapAsync(this)
     }
 
-    override fun getCameraDataModel(): Camera.CameraModel {
-        return cameraDataModel
-    }
+    override fun getMapDataModel() = ExtendedMapDataModel
```

Comment:
bez `.INSTANCE`?

Reply:
ano, v kotlinu se to takto zapisuje. jen když to voláš z Javy, tak tam dáváš to `.INSTANCE`. Je to divné, ale je to tak.